jenkins/build/Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+def PIPELINE_NAME = "build"
+
+def LIB_NAME= "identity-kit"
+def LIB_REPO = "https://github.com/bcgov/identity-kit-configurations.git"
+def LIB_BRANCH = "master"
```

Comment:
The default/main branch of this repository is `main`.  All branch references will need to be updated accordingly.